CMakeLists.txt

```
@@ -40,6 +40,61 @@ else()
   message(WARNING "Unknown compiler... recklessly proceeding without a version check")
 endif()
 
+set(LIBMONGOC_REQUIRED_VERSION 1.22.1) # TODO: update to 1.24.0 once released.
+set(LIBMONGOC_REQUIRED_ABI_VERSION 1.0)
+
+set(NEED_DOWNLOAD_C_DRIVER false CACHE INTERNAL "")
+if(TARGET mongoc_shared OR TARGET mongoc_static)
+  # If these targets exist, then libmongoc has already been included as a project
+  # sub-directory
+  message ("found libmongoc targets declared in current build scope; version not checked")
```

Always use a `message()` mode indicator

```suggestion
  message (STATUS "Found libmongoc targets declared in current build scope; version not checked")
```